ocean_lib/web3_internal/web3_overrides/test/test_contract.py

```diff
@@ -9,16 +9,16 @@
 from ocean_lib.web3_internal.web3_overrides.contract import CustomContractFunction
 
 
-def test_main(web3, dtfactory_address):
+def test_main(web3, config, dtfactory_address):
```

Comment:
Instead of taking in config which is big, just take on block_confirmations

Reply:
I think passing the config to unit tests is okay. Many unit tests do this. In this case `config` refers to the `config()` pytest fixture.

Passing only block_confirmations would require the creation of a new `block_confirmations()` pytest fixture. Please let me know if I should proceed with this change anyway.